exampleCourse/questions/demo/annotated/engCircuit/question.html

```diff
@@ -2,7 +2,7 @@
 
 <p>Determine the {{params.ask}} of the circuit below. Make sure to include a magnitude and unit in your answer.</p>
 
-<p><pl-figure file-name="figure.svg" type="dynamic" width="100%" alt="{{params.alt}}"></pl-figure></p>
+<p><pl-figure file-name="figure.svg" type="dynamic" alt="{{params.alt}}"></pl-figure></p>
```

Comment:
Nit, remove the invalid `<p>` wrapper from here.

Reply:
These will be bulk fixed up in a follow up PR too. c028248